src/Particular.CodeRules/Particular.ruleset

```diff
@@ -230,4 +230,54 @@
     <Rule Id="CA5403" Action="None" />             <!-- Do not hard-code certificate -->
     <Rule Id="CA9999" Action="None" />             <!-- Analyzer version mismatch -->
   </Rules>
-</RuleSet>
\ No newline at end of file
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <!-- rules we may want to enforce eventually -->
+    <Rule Id="SA0001" Action="None" />             <!-- XML comment analysis disabled -->
+    <Rule Id="SA1005" Action="None" />             <!-- Single line comments should begin with single space -->
+    <Rule Id="SA1009" Action="None" />             <!-- Closing parenthesis should be spaced correctly -->
+    <Rule Id="SA1012" Action="None" />             <!-- Opening braces should be spaced correctly -->
+    <Rule Id="SA1013" Action="None" />             <!-- Closing braces should be spaced correctly -->
+    <Rule Id="SA1028" Action="None" />             <!-- Code should not contain trailing whitespace -->
+    <Rule Id="SA1121" Action="None" />             <!-- Use built-in type alias -->
+    <Rule Id="SA1127" Action="None" />             <!-- Generic type constraints should be on their own line -->
+    <Rule Id="SA1128" Action="None" />             <!-- Put constructor initializers on their own line -->
+    <Rule Id="SA1135" Action="None" />             <!-- Using directives should be qualified -->
+    <Rule Id="SA1201" Action="None" />             <!-- Elements should appear in the correct order -->
```

Comment:
As far as I'm aware the previous Resharper template always included ordering requirements. I think it greatly improves the readability of the code when all classes are ordered the same. Of course one can always argue which order is the best. I personally do not care as long as things are ordered instead of wasting time in every downstream where every opinionated author of a PR places things in different order because the believe their order is right :)

Reply:
The TF agree with that. However, it's a *huge* undertaking. That's why this rule is in the section "rules we may want to enforce eventually".

Reply:
BTW, the reason we are not enforcing it now is that [the order in StyleCop is fixed and cannot be changed](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md), and it's totally different to what we currently have. It matches what seems to be used in most open source projects, which is totally different what we have.

Reply:
Makes sense